Lib/enum.py

```diff
@@ -148,6 +147,14 @@ def __new__(metacls, cls, bases, classdict):
 
         # save enum items into separate mapping so they don't get baked into
         # the new class
+        for i,name in enumerate(classdict._member_names):
+            value = classdict[name]
+            if isinstance(value, auto):
+                if value.value == _auto_null:
+                    value.value = classdict._generate_next_value(name, 1, len(classdict._member_names[:i]), classdict._last_values[:i])
+                super(type(classdict), classdict).__setitem__(name, value.value)
+                classdict._last_values[i] = value.value
+        super(type(classdict), classdict).__setitem__('_auto_called', True)
```

Comment:
Setting `_auto_called` here has no effect.

Reply:
```
class Dupes(Enum):
    first = primero = auto()
```
In this case,`primero` is not `_auto_null`.

Reply:
Ah, you're right.  Thanks.